app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.scss

```diff
@@ -49,7 +49,7 @@
   .icon {
     border-radius: 50%;
     border: 3px solid $color-pf-black-300;
-    padding: 11px;
+    padding: 12px;
```

Comment:
what's the reasoning behind changing the padding?

Reply:
Nice find. Padding and height has been added to the `.icon-container`. Now all step icons are 55x55 pixels.